test/views/start_test.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:cie_team1/views/start.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/material.dart';
+
+@Timeout(const Duration(seconds: 20))
+void main() {
+
+  testWidgets('1 widgetTest', (WidgetTester tester) async {
+    // Tells the tester to build a UI based on the widget tree passed to it
+    await tester.pumpWidget(
+      new StatefulBuilder(
+        builder: (BuildContext context, StateSetter setState) {
+          return new MaterialApp(
+            home: new Material(
+              child: new Center(
+                  child: new WelcomePage()
+              ),
+            ),
+          );
+        },
+      ),
+    );
+
+
+    final Iterable<Widget> listOfWidgets = tester.allWidgets;
+
+    int counter = 0;
+    for (Widget widget in listOfWidgets) {
+      if (widget is Text) {
+        if (counter == 0) {
+          expect(widget.data, 'Courses in English');
+          counter++;
+        } else if (counter == 1) {
+          expect(widget.data, 'Start');
+          counter++;
+        }
+      }
+    }
+
+    await tester.tap(find.text('Courses in English'));
```

Comment:
Can you explain, what this line of code does?
Am I right, that it will search for 'Courses in English' and then tap/click it?

Reply:
yeah you are right.

My intention was that this button should be clickable

Reply:
Hmm... As far as I know there should be a 'Start' Button.

But that should be a easy fix and also this test is not expecting anything after that, so it should be no problem...

Reply:
sorry was to fast in appriving tobi

Reply:
I think for the first step we can remove this line and maybe add some integration tests later?
@EduardBartolovic what do you mean?